config/vm.args

```diff
@@ -1,6 +1,5 @@
 -sname epoch@localhost
 
--setcookie epoch_cookie
```

Comment:
I hope it's planned

Reply:
It is

Reply:
Alternative:
From [OTP 20.1 doc](http://erlang.org/documentation/doc-9.1/lib/kernel-5.4/doc/html/kernel_app.html): `{inet_dist_use_interface, ip_address()}`.
Used [here](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/src/inet_tcp_dist.erl#L117).

Reply:
> Alternative:
> From [OTP 20.1 doc](http://erlang.org/documentation/doc-9.1/lib/kernel-5.4/doc/html/kernel_app.html): `{inet_dist_use_interface, ip_address()}`.
> Used [here](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/src/inet_tcp_dist.erl#L117).

`net_kernel` checks `proto_dist` init argument and if none it uses `inet_tcp_dist` ([doc](https://github.com/erlang/otp/blob/OTP-20.1/erts/doc/src/erl.xml#L439-L444), [code](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/src/net_kernel.erl#L1380-L1384)).

`net_kernel` listens calling [`inet_tcp_dist:listen(Name)`](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/src/net_kernel.erl#L1399-L1400) that calls internal `inet_tcp_dist:do_listen` [with `Driver` `inet_tcp` (hardcoded)](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/src/inet_tcp_dist.erl#L71-L75). This, if `application:get_env(kernel, inet_dist_use_interface)` returns `{ok, Ip}`, adds `{ip, Ip}` to options passed to `Driver:listen(First, Options)` i.e. [`gen_tcp:listen/2`](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/src/inet_tcp.erl#L140), that in turn [parses option `{ip, _}`](https://github.com/erlang/otp/blob/OTP-20.1/lib/kernel/src/inet.erl#L802).

So option `inet_dist_use_interface` seems hooked properly.

Reply:
Coded as https://github.com/aeternity/epoch/pull/1859 - testing